Bottom: dbca0743a67790bab7d4bf125af3e5e1f56afacb
Top:    d184a07b1c2d046ae5d4ab7f18c255cd11dbaf47
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-03-05 14:43:54 -0800

Refresh of xfer-cnt

---

diff --git a/examples/common.c b/examples/common.c
index 54339d5..aac679e 100644
--- a/examples/common.c
+++ b/examples/common.c
@@ -118,7 +118,7 @@ void cnt_str(char *str, size_t ssize, long long cnt)
 int size_to_count(int size)
 {
 	if (size >= (1 << 20))
-		return 250;
+		return 300;
 	else if (size >= (1 << 16))
 		return 1000;
 	else if (size >= (1 << 10))
